installer/mlflow/description.yaml

```diff
@@ -1,11 +1,12 @@
 name: mlflow
 description: |
   MLFlow is an open source platform specialized in tracking ML experiments, and packaging and deploying ML models.
-namespace:
+namespace: fuseml-workflows
```

Comment:
why is this on `fuseml-workflows` namespace?

Reply:
This is the default namespace we are installing `mlflow`. I think you implemented it that way originally

Reply:
aha, then you mistyped it. The workspace name is actually `fuseml-workloads`. Also, we might want to move it out of that namespace, but I guess this is for another PR.

Reply:
Ah, you are right!

As for having extra namespace for mlflow, I agree, but it might require some further changes. So indeed, let's leave it for later.